spec/amp-framework-hosting.md

```diff
@@ -0,0 +1,263 @@
+
+```
+<meta name="runtime-host" content="https://example.com/amp">
+<script async src="https://example.com/amp/v0.js"></script>
+<script async custom-element="amp-geo" src="https://example.com/amp/v0/amp-geo-0.1.js"></script>
+```
+
+### Metadata
+
+The AMP Project hosts a metadata endpoint at [cdn.ampproject.org/rtv/metadata](https://cdn.ampproject.org/rtv/metadata) that returns information on current releases. Hosting this endpoint yourself is optional, but may be useful if you use [AMP Toolbox](https://github.com/ampproject/amp-toolbox):
+
+- [@ampproject/toolbox-runtime-version](https://github.com/ampproject/amp-toolbox/tree/master/packages/runtime-version) uses the data to determine the latest AMP framework version available.
+- [@ampproject/toolbox-optimizer](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer) uses the data to identify the boilerplate CSS that should be included in optimized AMP pages.
+- [@ampproject/toolbox-runtime-fetch](https://github.com/ampproject/amp-toolbox/tree/master/packages/runtime-fetch) uses the data to identify an rtv-specific path from which the framework should be downloaded.
+
+Consider the following sample from [cdn.ampproject.org/rtv/metadata](https://cdn.ampproject.org/rtv/metadata), most JSON properties are optional:
+
+```
+{
+  "ampRuntimeVersion": "012002192257490",
+  "ampCssUrl": "https://cdn.ampproject.org/rtv/012002192257490/v0.css",
+  "canaryPercentage": "0.005",
+  "diversions": ["002002251816300", "032002251816300", "022002192257490"],
+  "ltsRuntimeVersion": "012002191527100",
+  "ltsCssUrl": "https://cdn.ampproject.org/rtv/012002191527100/v0.css"
+}
+```
+
+The properties are defined as follows:
+
+- `ampRuntimeVersion` (required) is the current production runtime version of the AMP framework.
+- `ampCssUrl` (optional) is a URL to the boilerplate CSS for the current production runtime version.
+- `canaryPercentage` (optional) indicates the fraction of users who receive the current canary runtime version of the AMP framework instead of the current production runtime version.
+- `diversions` (optional) lists active non-production runtime versions (canary and experiments).
```

Comment:
`(canary and experiments)` → `(e.g., Beta, Experimental, Control. See [#versioning](Versioning) for context)`

Reply:
Done